plugins/flytekit-dbt/flytekitplugins/dbt/util.py

```diff
@@ -0,0 +1,41 @@
+import json
+import logging
+import subprocess
+from typing import List
+
+
+def run_cli(cmd: List[str]) -> (int, List[str]):
```

Comment:
you can use - 
https://github.com/flyteorg/flytekit/blob/2ca5607d6b313fa9d6aeb88a807febcefcea7cdb/flytekit/tools/subprocess.py